### PR TITLE
Adding a note #1596

### PR DESCRIPTION
**Pull request by j0rt3g4**
Adding an important note to consideration of authors. #1596 

Copy review @nenonix @JohanFreelancer9 is required on this one guys. Thank you in advance!

### CODE REVIEW

File: OneDrive/control-access-to-mobile-app-features.md [hunk 1]
```diff
@@ -22,10 +22,13 @@ description: "Learn how to create a mobile application management policy for the
 
 # Control access to features in the OneDrive and SharePoint mobile apps
 
-If your organization has Microsoft Intune or Enterprise Mobility + Security, you can use the OneDrive admin center to create a global policy that manages the OneDrive and SharePoint mobile apps for Android and iOS. This policy only applies to users in your organization who are licensed for Microsoft Intune or Enterprise Mobility + Security.
+If your organization has **Microsoft Intune** or **Enterprise Mobility + Security**, you can use the OneDrive admin center to create a global policy that manages the OneDrive and SharePoint mobile apps for Android and iOS. This policy only applies to users in your organization who are licensed for Microsoft Intune or Enterprise Mobility + Security.
   
-> [!NOTE]
-> Your admin account needs to have an Intune license assigned to it for you to change the mobile application management settings in the OneDrive admin center. 
+ >[!NOTE]
```
Comment (JohanFreelancer9) [inline OneDrive/control-access-to-mobile-app-features.md]:
```suggestion
 > [!NOTE]
```

File: OneDrive/control-access-to-mobile-app-features.md [hunk 2]
```diff
@@ -22,10 +22,13 @@ description: "Learn how to create a mobile application management policy for the
...
+ >Your admin account needs to have an Intune license assigned to it for you to change the mobile application management settings in the OneDrive admin center.
```
Comment (JohanFreelancer9) [inline OneDrive/control-access-to-mobile-app-features.md]:
```suggestion
 >Your admin account needs to have an Intune license assigned to it for you to change the mobile application management settings in the OneDrive admin center. 
```

File: OneDrive/control-access-to-mobile-app-features.md [hunk 3]
```diff
@@ -22,10 +22,13 @@ description: "Learn how to create a mobile application management policy for the
...
 
-If your organization has Microsoft Intune or Enterprise Mobility + Security, you can use the OneDrive admin center to create a global policy that manages the OneDrive and SharePoint mobile apps for Android and iOS. This policy only applies to users in your organization who are licensed for Microsoft Intune or Enterprise Mobility + Security.
+If your organization has **Microsoft Intune** or **Enterprise Mobility + Security**, you can use the OneDrive admin center to create a global policy that manages the OneDrive and SharePoint mobile apps for Android and iOS. This policy only applies to users in your organization who are licensed for Microsoft Intune or Enterprise Mobility + Security.
   
-> [!NOTE]
-> Your admin account needs to have an Intune license assigned to it for you to change the mobile application management settings in the OneDrive admin center. 
+ >[!NOTE]
+ >Your admin account needs to have an Intune license assigned to it for you to change the mobile application management settings in the OneDrive admin center. 
+
+ >[!NOTE]
```
Comment (JohanFreelancer9) [inline OneDrive/control-access-to-mobile-app-features.md]:
```suggestion
 > [!NOTE]
```

File: OneDrive/control-access-to-mobile-app-features.md [hunk 4]
```diff
@@ -22,10 +22,13 @@ description: "Learn how to create a mobile application management policy for the
...
+ > In the case that you disable a policy, this won't remove it from Intune.
```
Comment (JohanFreelancer9) [inline OneDrive/control-access-to-mobile-app-features.md]:
```suggestion
 > Disabling a policy won't remove it from Intune.
```

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**